### DISCUSSION TITLE
ModuleID(...) not found.

### DISCUSSION THREAD

**Discussion by hkhc** [Splitties/refreshVersions#331] [General]
I don't know if it is a bug or I misunderstand something. So I put it here.

I am developing a plugin (well actually two, in the same project). I publish them to MavenLocal, so I have directories in `.m2` directory like this:

```
~/.m2/repository/io/hkhc/jarbird/io.hkhc.jarbird.gradle.plugin
~/.m2/repository/io/hkhc/jarbird-android/io.hkhc.jarbird-android.gradle.plugin
```

They are both usable with my testing project.  Then I introduced refreshVersions in my testing project. The `versions.properties ` consists of this:

```
plugin.io.hkhc.jarbird=0.5.1
plugin.io.hkhc.jarbird-android=0.5.1
```

The plugins are applied in two subprojects of the testing project respectively. It works well to resolve the plugins and run the project successfully.

So here is the problem. When I run `./gradlew refreshVersions` with the testing project. I got the following:

```
Execution failed for task ':refreshVersions'.
> ModuleId(group=io.hkhc.jarbird-android, name=io.hkhc.jarbird-android.gradle.plugin) not found. Searched the following repositories:
  - file:/Users/hc/.m2/repository/
  - https://repo.maven.apache.org/maven2/
  - https://dl.google.com/dl/android/maven2/
  - https://jcenter.bintray.com/
  - https://plugins.gradle.org/m2/
```

This error only occurs with the `io.hkhc.jarbird-android` plugin. There is no error if I use `io.hkhc.jarbird` only.

So is there any condition that make the plugin workable with refreshVersions?

**Comment by LouisCAD** [comment]
Hello, it's a known issue regarding maven local support that has been fixed but not released yet.
